src/components/claiming/ClaimingInfo.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="main">
+    <div class="header">
+      <div class="main-col border">
+        <p class="title">PARTICIPATION</p>
+        <p class="sub-title">${{ participation }}</p>
+      </div>
+      <div class="main-col border">
+        <p class="title">TOTAL TOKENS ALLOCATION</p>
+        <p class="sub-title">{{ amount }} <span class="currency">WIT</span></p>
+      </div>
+      <div class="main-col">
+        <p class="title">LOCK-UPS</p>
+        <p class="sub-title" v-show="vesting.length >= 1">Yes</p>
+        <p class="sub-title" v-show="vesting.length <= 1">No</p>
+      </div>
+    </div>
+    <div class="list-data">
+      <div class="list-row" v-for="step in vesting" :key="step.date.getTime()">
+        <p class="data type">{{ type }}</p>
+        <p class="data date">{{ changeDateFormat(step.date) }}</p>
+        <p class="data amount">{{ step.amount }}<span class="currency"> WIT</span></p>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapState } from 'vuex'
+import { calculateVesting, changeDateFormat } from '@/utils'
+export default {
+  name: 'ClaimingInfo',
+  computed: {
+    ...mapState({
+      vestingInfo: state => state.wallet.claimingFileInfo.info.data.vesting,
+      genesisDate: state => state.wallet.claimingFileInfo.info.data.genesisDate,
+      amount: state => state.wallet.claimingFileInfo.info.data.wit,
+      participation: state => state.wallet.claimingFileInfo.info.data.usd,
+    }),
+    type() {
+      const genesis = this.vesting.lenght <= 1
+      const delay = this.vestingInfo.delay > 0 && !genesis
+      return genesis ? 'GENESIS' : delay ? 'DELAY' : 'LOCK-UP'
+    },
+    vesting() {
+      const vesting = calculateVesting(this.vestingInfo, this.amount, this.genesisDate)
+      this.$store.commit('setComputedVesting', vesting)
```

Comment:
Move this vuex call to a watcher that will be called when vesting returns a value